### PR TITLE
fix: standardizing style for action links

### LINKED ISSUE #5176: Review of how we tag links and buttons
Miguel mentioned that the same links are tagged differently e.g. action links. The main problem is our no-breaking-change rule: that we can't easily make style changes when we notice something or want to change something. I want to understand the issues and make a list for 2.0 release so we can streamline and provide an improved styling system for customers.

### PR DESCRIPTION
**Pull request by developpeurweb**
fixes GH-5176

### CODE REVIEW

**Review by morganm58 — DISMISSED** [review]

**Review by hackergil — DISMISSED** [review]

File: projects/storefrontlib/src/cms-components/cart/add-to-wishlist/add-to-wish-list.component.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
             [disabled]="loading$ | async"
           >
             <cx-icon [type]="iconTypes.HEART"></cx-icon>
-            <span class="button-text">{{
+            <span class="button-text cx-action-link">{{
```
Comment (tobi-or-not-tobi) [inline projects/storefrontlib/src/cms-components/cart/add-to-wishlist/add-to-wish-list.component.html]: Generally speaking I'd prefer the action link class on the button, as it's better related imho. We could have also avoided the `span` element I guess. I see you do that in other occasions in this PR, but not sure if there's a common pattern in the code base. I leave it to you to decide.

**Review by tobi-or-not-tobi — DISMISSED** [review]

File: projects/storefrontlib/src/cms-components/cart/add-to-wishlist/add-to-wish-list.component.html (hunk 1 above)
Comment (developpeurweb) [reply projects/storefrontlib/src/cms-components/cart/add-to-wishlist/add-to-wish-list.component.html]: @tobi-or-not-tobi indeed! and fixed, can you please re-approve.

File: projects/storefrontlib/src/cms-components/product/product-list/product-facet-navigation/product-facet-navigation.component.html [hunk 2]
```diff
@@ -7,75 +7,7 @@
...
-              >
-                {{ 'productList.showMore' | cxTranslate }}
-              </li>
-            </ul>
-          </ng-container>
-        </div>
-      </ng-container>
-    </ng-container>
-  </div>
+<cx-active-facets></cx-active-facets>
```
Comment (tobi-or-not-tobi) [inline projects/storefrontlib/src/cms-components/product/product-list/product-facet-navigation/product-facet-navigation.component.html]: This looks wrong. The facet navigation was refactored heavily, you probably need to merge it but not override anything.

**Review by tobi-or-not-tobi — CHANGES_REQUESTED** [review]

File: projects/storefrontlib/src/cms-components/product/product-list/product-facet-navigation/product-facet-navigation.component.html (hunk 2 above)
Comment (developpeurweb) [reply projects/storefrontlib/src/cms-components/product/product-list/product-facet-navigation/product-facet-navigation.component.html]: @tobi-or-not-tobi I'm aware of the changes, if i compare this file with `develop` and with the branch where you made those changes, all 3 files look the same.

Comment (developpeurweb) [reply projects/storefrontlib/src/cms-components/product/product-list/product-facet-navigation/product-facet-navigation.component.html]: Screen Shot 2020-04-20 at 9 38 55 AM

**Review by tobi-or-not-tobi — APPROVED** [review]

**Pull request merged**